DllImportGenerator/DllImportGenerator/Resources.resx

```diff
@@ -159,6 +159,12 @@
   <data name="GetPinnableReferenceShouldSupportAllocatingMarshallingFallbackMessage" xml:space="preserve">
     <value>Type '{0}' has a 'GetPinnableReference' method but its native type does not support marshalling in scenarios where pinning is impossible</value>
   </data>
+  <data name="MarshallingCharAsCharSetAnsiNotSupported" xml:space="preserve">
+    <value>Marshalling char with CharSet.Ansi is not supported. Instead, manually convert the char type to a single byte and pass to the source-generated P/Invoke.</value>
+  </data>
+  <data name="MarshallingCharAsCharSetAutoNotSupported" xml:space="preserve">
+    <value>Marshalling char with CharSet.Auto is not supported. Instead, manually convert the char type to a single byte and pass to the source-generated P/Invoke.</value>
```

Comment:
`a single byte` -> the desired byte representation? Not sure what exactly that message should be.